tests/EnvJsonTest.php

```
@@ -4,56 +4,168 @@
+use Koriym\EnvJson\Exception\InvalidJsonSchemaException;
 use Koriym\EnvJson\Exception\SchemaFileNotFoundException;
+use Koriym\EnvJson\Exception\SchemaFileNotReadableException;
 use PHPUnit\Framework\TestCase;
+use stdClass;
 
 use function getenv;
 use function putenv;
 
 class EnvJsonTest extends TestCase
 {
+    private array $originalEnv = [];
```

PHPStan (2.0.3)

Property Koriym\EnvJson\EnvJsonTest::$originalEnv type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)